extensions/extensions-helm/aissemble-airflow-chart/values.template.yaml

```diff
@@ -4,8 +4,11 @@
 airflow:
   airflowVersion: 2.6.2
 
+
+  # NB: 1.7.0 is the last version to include airflow support. It is being frozen there until it is removed or pulled
+  # into an extension project.  TODO in https://github.com/boozallen/aissemble/issues/227
   defaultAirflowRepository: ghcr.io/boozallen/aissemble-airflow
-  defaultAirflowTag: @version.aissemble@
+  defaultAirflowTag: 1.7.0
```

Comment:
A: Might make sense to remove the template processing wholesale since the version number was the only thing we were using it for.

Reply:
I'm certainly open to that - I assumed there all followed a default pattern.

Reply:
Done